src/components/formik-forms/formik-select.jsx

```diff
@@ -37,6 +37,7 @@ const FormikSelect = ({
             </Field>
             {error && (
                 <ValidationMessage
+                    isError
```

Comment:
probably a silly react question, but where is this defined/where does the value come from?

Reply:
if you don't set any value on a prop like this, it makes its value be `true`. I think this is an HTML spec thing.